source/features/search/ui/TransactionSearchResult.tsx

```diff
@@ -17,6 +17,9 @@ export const TransactionSearchResult = () => {
   const search = useSearchFeature();
   const networkInfo = useNetworkInfoFeature();
   const navigation = useNavigationFeature();
+  const [isShowingUnmoderatedData, setisShowingUnmoderatedData] = useState(
+    true
+  );
```

Comment:
Is there a specific reason why the state for moderated data is kept in the search result and not directly in the `TransactionInfo` component?

Reply:
I're right, overcomplicating without a reason. I have simplified it now.